web/app.js

```diff
@@ -396,15 +396,15 @@ var PDFViewerApplication = {
   },
 
   get pagesCount() {
-    return this.pdfDocument.numPages;
+    return this.pdfDocument ? this.pdfDocument.numPages : 0;
   },
 
   set page(val) {
-    this.pdfLinkService.page = val;
+    this.pdfViewer.currentPageNumber = val;
   },
 
   get page() { // TODO remove
-    return this.pdfLinkService.page;
+    return this.pdfViewer.currentPageNumber;
```

Comment:
Now that this does not use the link service anymore, do we still need the `TODO remove` comment one line above? I don't see why this getter needs to go anymore.